element_array_ephys/ephys_chronic.py

```diff
@@ -106,6 +131,63 @@ class ProbeInsertion(dj.Manual):
     insertion_datetime=null: datetime
     """
 
+    @classmethod
+    def auto_generate_entries(cls, session_key):
```

Comment:
`auto_generate_entries` method for `ProbeInsertion` is not applicable for `ephys_chronic`, we can remove this method entirely here